src/js/datamaps.js

```diff
@@ -330,10 +330,24 @@
 
   function handleLabels ( layer, options ) {
     var self = this;
-    options = options || {};
-    var labelStartCoodinates = this.projection([-67.707617, 42.722131]);
+    var defaultOptions = {
+      fontFamily: "Verdana",
+      fontSize: 10,
+      passLabelMouseEvents: true,
+      labelColor: "#000",
+      labelStartCoordinates: [-67.707617, 42.722131],
+      labelSpacing: 2,
+      lineColor: "#000",
+      lineWidth: 1,
+      popupOnLabelHover: true,
+      smallSubunits: ["VT", "NH", "MA", "RI", "CT", "NJ", "DE", "MD", "DC"],
+      smallSubunitsOnly: false
+    };
+    options = defaults(options, defaultOptions);
```

Comment:
Can you add a check here for an `undefined` `options`? (Like it was in the original line 333).

I get an error when I just call `map.labels()`